### PR TITLE
Clean python2

### PR DESCRIPTION
**Pull request by mosfet80**
Thank you for creating a pull request to contribute to FreeCAD! Place an "X" in between the brackets below to "check off" to confirm that you have satisfied the requirement, or ask for help in the [FreeCAD forum](https://forum.freecadweb.org/viewforum.php?f=10) if there is something you don't understand.

- [ ]  Your Pull Request meets the requirements outlined in section 5 of [CONTRIBUTING.md](https://github.com/FreeCAD/FreeCAD/blob/master/CONTRIBUTING.md) for a Valid PR

Please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [1.0 Changelog Forum Thread](https://forum.freecad.org/viewtopic.php?f=10&t=69438).

---

### CODE REVIEW

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738">pipeline status</a> for feature branch [PR_8738](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738). Pipeline [796283176](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/796283176) was triggered at [d9d57ae](https://github.com/FreeCAD/FreeCAD/pull/8738/commits/d9d57ae70b72b2143e56b780e0159c929db80ad1). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by chennes** [comment]
The stuff in the Addon Manager is a list of old addons that only support Python2 -- it provides an error message to a user looking for those addons, since they cannot be run in current versions of FreeCAD. That needs to stay for the time being.

File: src/Mod/Fem/feminout/importPyMesh.py [hunk 1]
```diff
@@ -38,12 +38,7 @@
 # names are fix given from FreeCAD, these methods are called from FreeCAD
 # they are set in FEM modules Init.py
 
-if open.__module__ == "__builtin__":
-    # because we'll redefine open below (Python2)
-    pyopen = open
-elif open.__module__ == "io":
-    # because we'll redefine open below (Python3)
-    pyopen = open
+ipyopen = open
```
Comment (chennes) [inline src/Mod/Fem/feminout/importPyMesh.py:41]: Typo (I assume).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738">pipeline status</a> for feature branch [PR_8738](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738). Pipeline [796303550](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/796303550) was triggered at [540089c](https://github.com/FreeCAD/FreeCAD/pull/8738/commits/540089c39bbe0eb40f0a564b6352293043286d65). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by freecadci** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738">pipeline status</a> for feature branch [PR_8738](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_8738). Pipeline [796308618](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/796308618) was triggered at [fec52b6](https://github.com/FreeCAD/FreeCAD/pull/8738/commits/fec52b6178ff1289906c2eec78f9afeb3ebc074b). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Pull request merged**